pinot-broker/pom.xml

```diff
@@ -106,6 +106,26 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>io.swagger</groupId>
```

Comment:
double check if you need jackson dependencies. Changes here don't seem to introduce jackson requirements